examples/src/demos/Raycast.js

```diff
@@ -0,0 +1,135 @@
+import React, { Suspense, useState, useRef, useEffect } from 'react'
+import { Canvas, useFrame, useThree, Dom, extend } from 'react-three-fiber'
+import { Physics, useSphere, useBox, useRaycastAll } from 'use-cannon'
+import { Vector3 } from 'three'
+import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls'
+
+extend({ OrbitControls })
+
+function Plane(props) {
+  return (
+    <group {...props}>
+      <mesh>
+        <planeBufferGeometry attach="geometry" args={[8, 8]} />
+        <meshBasicMaterial attach="material" color="#FFD3A5" />
+      </mesh>
+      <mesh receiveShadow>
+        <planeBufferGeometry attach="geometry" args={[8, 8]} />
+        <shadowMaterial attach="material" color="#f8cd7e" />
+      </mesh>
+    </group>
+  )
+}
+
+function Sphere({ position }) {
+  const [ref, api] = useSphere(() => ({ type: 'Static', position, args: 0.5 }))
+  useFrame(({ clock: { elapsedTime } }) => {
+    api.position.set(Math.sin(elapsedTime / 2) * 2, position[1], position[2])
+  })
+  return (
+    <mesh castShadow ref={ref}>
+      <sphereBufferGeometry attach="geometry" args={[0.5, 32, 32]} />
+      <meshNormalMaterial attach="material" />
+    </mesh>
+  )
+}
+
+function Cube({ position }) {
+  const [ref, api] = useBox(() => ({ type: 'Static', position }))
```

Comment:
Same issue as described in the comments, default cube size is 1, but displayed geometry has edge lengths of 0.5. Great work otherwise :)